frontend/src/layouts/pageSectionsWithNav.ts

```diff
@@ -24,7 +26,8 @@ export function pageSectionsWithNav({
       <div
         class=${clsx(
           tw`flex flex-1 flex-col gap-2`,
-          sticky && tw`lg:sticky lg:top-2 lg:self-start`,
+          sticky &&
+            tw`lg:sticky lg:self-start ${stickyTopClassname || tw`lg:top-2`}`,
```

Comment:
I'm not sure how well Tailwind (& its various formatting/intellisense plugins) support string interpolation, but clsx supports nesting
```suggestion
            [tw`lg:sticky lg:self-start`, stickyTopClassname || tw`lg:top-2`],
```